### ISSUE TITLE
Cannot install yakuake-frameworks-git (dependency not found)

### ISSUE THREAD

**Issue by Jiehong** [aurapm/aura#297]
When trying to install `yakuake-frameworks-git` from AUR, aura does not find a dependency:

``` sh
$ sudo aura -A yakuake-frameworks-git
aura >>= Determining dependencies...
aura >>= Dependency checking failed for these reasons:
The dependency `libkonq-frameworks-git` could not be found. You may need to search for a package to satisfy it.
```

However, this dependency is present in AUR:

``` sh
$ sudo aura -As libkonq-frameworks-git
aur/libkonq-frameworks-git r19537.fda1e2c-1 (2)
    KDE libraries for the basic desktop applications
```

Some debug information:

``` sh
$ sudo aura --auradebug -A yakuake-frameworks-git 
User              => root
True User         => xxx
Build User        => xxx
Using Sudo?       => Yes!
Pacman Flags      => 
Other Flags       => Debug AURInstall
Other Input       => yakuake-frameworks-git
Language          => English
Pacman Command    => pacman
Editor            => vi
$CARCH            => x86_64
Ignored Pkgs      => 
Build Path        => /var/cache/pacman/pkg/
Pkg Cache Path    => /var/cache/pacman/pkg/
Log File Path     => /var/log/pacman.log
Quiet?            => No.
Silent Building?  => Yes!
Must Confirm?     => Yes!
Needed only?      => No.
PKGBUILD editing? => No.
Diff PKGBUILDs?   => No.
Rebuild Devel?    => No.
Use Customizepkg? => No.
Forego PowerPill? => No.
Keep source?      => No.
aura >>= Determining dependencies...
aura >>= Dependency checking failed for these reasons:
The dependency `libkonq-frameworks-git` could not be found. You may need to search for a package to satisfy it.
```

**Comment by fosskers** [comment]
`libkonq-framworks-git` seems to be a virtual / split package. Notice doing `-Ai` or `-Ap` doesn't work on it. I think the `aur` Haskell library needs an update to account for this.

**Comment by fosskers** [comment]
Getting `aura: Maybe.fromJust: Nothing` with this right now.

**Comment by fosskers** [comment]
The original package listed here doesn't exist anymore, but this problem is solved by my latest patches on the `aura-1.5` branch.
